lib/src/components/NxTab/__tests__/NxTab.test.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+import NxTab from '../NxTab';
+
+describe('NxTab', function () {
+  it('renders a tab with the expected class names', function () {
+    const component = shallow(<NxTab id="tab"/>);
+
+    expect(component).toHaveProp('aria-selected', 'false');
+    expect(component).not.toHaveClassName('active');
+  });
+
+  it('renders a tab with the expected class names when active={false}', function () {
+    jest.spyOn(React, 'useContext').mockImplementation(() => 'tab');
+    const component = shallow(<NxTab id="tab"/>);
```

Comment:
Rather than mocking useContext maybe the shallow rendered JSX here should include an actual context provider?

Reply:
6fe2aeb